docs/custom-webview-android.md

```diff
@@ -5,32 +5,41 @@ title: Custom WebView
 
 While the built-in web view has a lot of features, it is not possible to handle every use-case in React Native. You can, however, extend the web view with native code without forking React Native or duplicating all the existing web view code.
 
+:::info
+The React Native WebView component has been moved under `react-native-community/react-native-webview` as part of the [lean core effort](https://github.com/facebook/react-native/issues/23313).
```

Comment:
```suggestion
The React Native WebView component has been extracted to `react-native-webview` package as part of the [Lean Core effort](https://github.com/facebook/react-native/issues/23313).
```

Thanks for the updates! 👍 Just one more, small wording tweak.

Reply:
Done